tests/test_results_table.py

```diff
@@ -0,0 +1,119 @@
+import os
+import tempfile
+from unittest import TestCase
+
+from hdmf.common import HDF5IO, get_manager, get_hdf5io, EnumData
+from hdmf_ml import ResultsTable
+
+import numpy as np
+
+
+class ResultsTableTest(TestCase):
+
+    def setUp(self):
+        _, self.path = tempfile.mkstemp()
+
+    def tearDown(self):
+        if os.path.exists(self.path):
+            os.remove(self.path)
+
+    def get_hdf5io(self):
+        #return get_hdf5io(self.path, 'w')
+        return HDF5IO(self.path, 'w', manager=get_manager())
+
+
+    # add_tvt_split
+    # add_cv_split
+    # add_true_label
+    # add_predicted_probability
+    # add_predicted_class
+    # add_predicted_value
+    # add_cluster_label
+    # add_embedding
+    # add_topk_classes
+    # add_topk_probabilities
+
+    def test_ResultsTable_tvt_split(self):
+        rt = ResultsTable('foo', 'a test results table')
```

Comment:
```suggestion
        rt = ResultsTable(name='foo', description='a test results table')
```

I recommend using kwargs for constructors for data types. This may even be a warning in HDMF now.